app/services/plum_scanned_map_importer.rb

```diff
@@ -0,0 +1,521 @@
+# frozen_string_literal: true
+class PlumScannedMapImporter
+  attr_reader :id, :change_set_persister
+  attr_writer :all_collections, :id_cache
```

Comment:
This is perhaps following https://github.com/pulibrary/figgy/blob/88b4bafeeb7bfacc67af276e2238887058749157/app/services/plum_importer.rb#L4, but I does `id_cache` need to be a public mutator method?